engines/ospray/ispc/camera/PerspectiveCamera.h

```diff
@@ -45,13 +42,7 @@ struct OSPRAY_SDK_INTERFACE PerspectiveCamera : public Camera
     float focusDistance;
     bool architectural; // orient image plane to be parallel to 'up' and shift
                         // the lens
-    typedef enum {
-        OSP_STEREO_NONE,
-        OSP_STEREO_LEFT,
-        OSP_STEREO_RIGHT,
-        OSP_STEREO_SIDE_BY_SIDE
-    } StereoMode;
-    StereoMode stereoMode;
+    bool stereo;
```

Comment:
uninitialized variable (the ones above too!)

Reply:
The initialization for ospray objects to default values is done in `commit()`, where the default values are actually used if the given parameter does not exist. Check the cpp file for that.